server/server.js

```diff
@@ -1,90 +1,87 @@
-var express = require('express');
-var https = require('https');
-var http = require('http');
-var fs = require('fs');
-var pty = require('pty.js');
-var sys = require('sys')
-var exec = require('child_process').exec;
-var tmp = require('tmp');
-var path = require('path');
-var assert = require('assert');
-
-var Docker = require('dockerode');
-var docker = new Docker();
+const express = require('express');
+const http = require('http');
+const fs = require('fs');
+const pty = require('node-pty');
+const tmp = require('tmp');
+const path = require('path');
+
+const Docker = require('dockerode');
+const docker = new Docker();
 
 require('log-timestamp');
 
 // Setup the express app
-var app = express();
+const app = express();
 // Static file serving
-app.use("/", express.static("client"));
+app.use('/', express.static('client'));
 
 // Creating an HTTP server
-var server = http.createServer(app).listen(8080)
+const server = http.createServer(app).listen(7171);
```

Comment:
I changed port 8080 to 7171 simply because 8080 is used in many other projects, so this new port will hopefully avoid collisions with other projects the user wants to run on localhost (this happened to me, I typically will run a database API on 8080, which is the thing I want to target with fake traffic).